frontend/src/components/roadmap/sections/RoadmapWhatWereWorkingOn.tsx

```diff
@@ -11,36 +11,31 @@ export default function RoadmapWhatWereWorkingOn() {
 
   return (
     <RoadmapPageSection className={"bg-white"} title={t("title")}>
-      <div className="margin-top-1" />
       {contentItems.map((contentRows, contentRowsIdx) => (
         <div
-          className="grid-row"
+          className="grid-row grid-gap"
           key={`roadmap-what-were-working-on-${contentRowsIdx}`}
         >
           {contentRows.map((contentRowItem, contentRowItemIdx) => (
             <div
-              className="margin-bottom-3 tablet-lg:grid-col-6 tablet-lg:padding-right-5"
+              className="margin-bottom-4 tablet:grid-col-6"
```

Comment:
- Add margin (compensate for removed empty div above)
- Remove padding (use grid gap above instead)